src/commands/schema/cat.js

```diff
@@ -0,0 +1,46 @@
+const FaunaCommand = require("../../lib/fauna-command.js");
+const { Args } = require("@oclif/core");
+const fetch = require("node-fetch");
+
+class CatSchemaCommand extends FaunaCommand {
```

Comment:
i'm not sure if we wanted to keep this command. its kinda useful, but doesn't fit with the project management stuff all that well

Reply:
nvm we're keeping them